addon/components/select-2.js

```diff
@@ -373,6 +374,9 @@ var Select2Component = Ember.Component.extend({
       this.selectionChanged(data);
     }));
 
+    // add in the lower search placeholder if the value was provided (only on initialization)
+    if (this.get('searchPlaceholder')) { this.searchPlaceholderChanged(); }
```

Comment:
wanna use the same strategy as below if you're going to call it this?

Otherwise change the name to `setSearchPlaceholder`